speech_assets/SampleUtterances.txt

```diff
@@ -1,18 +1,36 @@
-AMAZON.StopIntent cancel
-AMAZON.StopIntent shut up
-AMAZON.StopIntent stop
```

Comment:
But I thought we do extend the intent?  It functions as both the 'standard' Amazon stop intent as well as the stop command for Kodi playback.